src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -1139,22 +1134,24 @@ public void attachSpeechControl(SpeechSynthesisControl control) {
     addListener(control.getName(), "publishSetMute");
     addListener(control.getName(), "publishReplaceWord");
   }
-
-  // Hacky way to get normalized code to work - inheritance not so helpful here
-  public SpeechSynthesisConfig getConfig(SpeechSynthesisConfig config) {
-    config.mute = mute;
-    config.blocking = blocking;
+  
+  
+  @Override
+  public ServiceConfig getConfig() {
+    SpeechSynthesisConfig c = (SpeechSynthesisConfig)config;

```

Comment:
why don't the specific services have their own subclass of speech synthesis config?  When loading this.. it seems like this will serialize out as a generic SpeechSynthesisConfig.. instead of the specific config type for the specific service...